app/Http/Controllers/Api/CityOpinionController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\CityOpinionRequest;
+use App\Models\CityOpinion;
+use Illuminate\Http\JsonResponse;
+
+class CityOpinionController extends Controller
+{
+    public function store(CityOpinionRequest $request): JsonResponse
+    {
+        $opinion = $request->validated();
+        $opinion["user_id"] = $request->user()->id;
+        CityOpinion::query()->create($opinion);
+
+        return response()->json([
+            "message" => __("Opinion added successfully."),
+        ]);
+    }
+
+    public function update(CityOpinionRequest $request, CityOpinion $cityOpinion): JsonResponse
+    {
+        $opinion = $request->validated();
+        $opinion["user_id"] = $request->user()->id;
+
+        $cityOpinion->update($opinion);
+
+        return response()->json([
+            "message" => __("Opinion edited successfully."),
+        ]);
+    }
+
+    public function destroy(CityOpinion $cityOpinion): JsonResponse
+    {
+        $cityOpinion->delete();
+
+        return response()->json([
+            "message" => __("Opinion removed successfully!"),
+        ]);
+    }
```

Comment:
Here too, I did not see in router applied policies.

Reply:
Are u going to add policies? Or maybe in separated PR?

Reply:
Was planning on including policies, after closing #169, since those are used there, and just merge main into this branch. Should i do it like that, or just duplicate them to here manually?